### PR TITLE
[WIP] Chipping away at modernizing the Readme.md

### PR DESCRIPTION
**Pull request by dannyrb**
@swederik working on that penance for my earlier accidental merge 🙏 

Let me know if anything in here seems off, or if you'd like to see anything added.

### CODE REVIEW

**Comment by dannyrb** [comment]
@swederik you can see the latest version at this URL:

https://github.com/dannyrb/cornerstoneTools/blob/aca0daa782453ce5c4b02b1fa942a160ece2e1f4/README.md

I'm curios to hear your thoughts on:

- Section ordering
- Moving backlog to "tracked issues" (and then removing the backlog section)
- Any other comments, concerns, or suggestions?

Hope I'm on the right track.

**Comment by dannyrb** [comment]
Really sorry about all of these "coveralls" notifications. The only good way to preview a layout is to make a commit and observe the changes.

Latest can be found here: https://github.com/dannyrb/cornerstoneTools/blob/b4e90565f22841b40e7fe30320ea3a9a3bc59eef/README.md

Bumped to 4 per row to allow for easier viewing on mobile.

**Comment by swederik** [comment]
This looks awesome to me! I also love how the git branch is marked as a chore :-)

Did you try Atom to preview the github-flavoured markdown? I also want to make coveralls less verbose.

**Comment by dannyrb** [comment]
Haha, my org uses `chore`, `feature`, and `bug` to prefix branches. They tend to align well with describing a single "story".

Good call out on using Atom! I actually use VS Code, but your comment reminded me to do some digging for a plugin: https://marketplace.visualstudio.com/items?itemName=bierner.markdown-preview-github-styles

👍  Thanks (:

I intend to:

- Add remaining GIFs
- Spellcheck
- Misc. tidying

And then I'll `@mention` you to see if we're good for a PR.

**Comment by dannyrb** [comment]
@swederik this _should_ be ready for review.

**Comment by swederik** [comment]
Looks good to me! Shall we merge it?

File: README.md [hunk 1]
```diff
@@ -4,110 +4,278 @@
...
 ================
 
-cornerstoneTools is a library built on top of [cornerstone](https://github.com/chafey/cornerstone) that provides
+`cornerstoneTools` is a library built on top of [cornerstone](https://github.com/chafey/cornerstone) that provides
 a set of common tools needed in medical imaging to work with images and stacks of images.
 
-View the [live examples](https://rawgithub.com/chafey/cornerstoneTools/master/examples/index.html) to see this
-library in action!
+* View [live examples](https://rawgithub.com/chafey/cornerstoneTools/master/examples/index.html) of individual tools
+* Take a peak at a [simple image viewer](http://chafey.github.io/cornerstoneDemo/) built on cornerstone
```
Comment (swederik) [inline README.md]: I guess you mean peek, not peak?

**Comment by dannyrb** [comment]
Going to :shipit: 
I'll likely make another quick PR to update image sources to use relative paths. Should make the change to an org easier.

Please feel free to queue up any documentation items you'd like to see. I'm starting to work on a hit list that prioritizes informing contributors/extenders.

**Pull request merged**